keras_nlp/models/distilbert/distilbert_preprocessing.py

```diff
@@ -84,66 +175,55 @@ class DistilBertPreprocessor(keras.layers.Layer):
     second_sentences = ["The fox tripped.", "Oh look, a whale."]
     labels = [1, 1]
     ds = tf.data.Dataset.from_tensor_slices(
-        ((first_sentences, second_sentences), labels))
+        (
+            (first_sentences, second_sentences), labels
+        )
+    )
     ds = ds.map(
         lambda x, y: (preprocessor(x), y),
         num_parallel_calls=tf.data.AUTOTUNE,
-    )
-    ```
+    )    ```
```

Comment:
I think the formatting on this line got messed up. Keep the triple backtick on a new line.

Reply:
Fixed!